package.json

```diff
@@ -85,6 +85,7 @@
     "lodash": "^4.17.11",
     "mocha": "^6.1.4",
     "nyc": "^14.1.1",
+    "obliterator": "github:forivall/obliterator#patch-1",
```

Comment:
Is there a PR for this we can wait for so I don't have to depend on a GitHub reference?

Reply:
Yeah, it did get merged. https://github.com/Yomguithereal/obliterator/pull/18. will update in a mo'